src/visit/traversal.rs

```diff
@@ -344,6 +344,18 @@ where
         topo
     }
 
+    /// Create a new `Topo` with initial nodes
```

Comment:
Continuing with the other comment, document the behaviour here.
```suggestion
    /// Create a new `Topo` with initial nodes.
    ///
    /// If any incoming edges to the initial nodes will be ignored.
```